go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/crypto v0.23.0
+	golang.org/x/crypto v0.31.0
```

Comment:
why not v0.32.0 if already available[1]?

---
[1] https://pkg.go.dev/golang.org/x/crypto